### ISSUE TITLE
IE9 - SCRIPT5: Access is denied

### ISSUE THREAD

**Issue by sebz** [vdurmont/userinfo-js#5]
Hi, 

I'm getting the following error when using IE9... (Yeah.. I know IE9 ...)
```
SCRIPT5: Access is denied.
 
userinfo.js, line 42 character 13
```

Thanks.

**Comment by sebz** [comment]
After investigating a little bit, it seems to be a crossorigin issue... I'm working on a fix.

**Comment by vdurmont** [comment]
Hey @sebz !
Thanks for raising the issue.
Let me know if you find something.

**Comment by vdurmont** [comment]
Fixed in pull request #6
